shim/src/main.rs

```diff
@@ -219,11 +219,11 @@ fn main() -> Result<()> {
         env::current_dir()?
     };
 
-    if clap_args.command.is_none() && clap_args.task.is_none() {
+    let mut args: Vec<_> = env::args().skip(1).collect();
```

Comment:
is `skip(1)` for `argv[0]` (i.e. the name of the process)?

Reply:
yep!